packages/investor-idle/src/idlecli.ts

```diff
@@ -0,0 +1,88 @@
+import { ethereum } from '@shapeshiftoss/chain-adapters'
+import { NativeAdapterArgs, NativeHDWallet } from '@shapeshiftoss/hdwallet-native'
+import * as unchained from '@shapeshiftoss/unchained-client'
+import dotenv from 'dotenv'
+
+import { bnOrZero } from './utils'
+import { IdleInvestor } from './IdleInvestor'
+
+dotenv.config()
+
+const { DEVICE_ID = 'device123', MNEMONIC } = process.env
```

Comment:
You'll need to create a `sample.env` or MNEMONIC will always be undefined here for use with CLI, see https://github.com/shapeshift/lib/blob/0a7bc55f2d6f7902a5f5af4d56e55a8bb00e4ebb/packages/investor-foxy/sample.env#L1

You can also add `DEVICE_ID` there instead of defaulting it here (this is taken from current implementations but is wrongly missing in the sample.env in said implementations)